src/clusterfuzz/_internal/bot/tasks/utasks/minimize_task.py

```diff
@@ -480,30 +467,27 @@ def utask_main(uworker_input):
     # We didn't crash at all. This might be a legitimately unreproducible
     # test case, so it will get marked as such after being retried on other
     # bots.
-    testcase = data_handler.get_testcase_by_id(uworker_input.testcase_id)
```

Comment:
I think while we are `put()ing` testcase in main, we should keep `get_testcase_by_id` because there can be concurrency problems.